lib/router.js

```diff
@@ -237,7 +243,6 @@ class Router {
                     throw new Error(`No operations exported by module ${loadPath}`);
                 }
                 if (!mod.globals) { mod.globals = {}; }
-                mod.globals.log = options.log;
```

Comment:
Why is this being removed? Some modules depend on it, AFAIK. Also, the YAML templates use it to log() things.

Reply:
Modules depend on options.log (and that needs to change, they should depend on options.logger.log), I don't know any instance of depending on options.globals.log.

I don't know any instances of yaml templates logging anything, I don't really think that will work even if you leave the log function in globals.